third_party/2and3/click/types.pyi

```diff
@@ -15,9 +15,11 @@ class CompositeParamType(ParamType):
 
 class Choice(ParamType):
     choices: Iterable[str]
+    case_sensitive: bool
     def __init__(self, choices: Iterable[str], case_sensitive: bool = ...) -> None: ...
 
 class DateTime(ParamType):
+    formats: Iterable[str]
     def __init__(self, formats: Optional[List[str]] = ...) -> None: ...
```

Comment:
As `len()` is called on `.formats`, this needs to be a `Sequence`. The `List` in `__init__` could also be changed. Another alternative is to just use `List` for the attribute to give additional guarantees to users.

Reply:
I see looking at the doc string for `DateTime.__init__`  it seems format should be either a Tuple or a List so the current type hint doesnt match this anyway.

So we either replace `Iterable[str]` and `List[str]` with `Union[List[str], Tuple[str]]` or as you suggested use `Sequence` which would be less strict I guess. Which do you prefer?

Reply:
`Sequence` is better I think.

Reply:
Ok I've updated the PR accordingly